projects/ngrx.io/content/guide/eslint-plugin/rules/signal-store-should-prefer-protected-state.md

```diff
@@ -0,0 +1,42 @@
+# signal-store-should-prefer-protected-state
+
+A Signal Store should prefer protected state.
+
+- **Type**: problem
```

Comment:
Covered that already above. I'd see this rule as a suggestion and definitely not fixable.